src/modules/emanation/emanation-dialog.ts

```diff
@@ -26,10 +27,10 @@ export class EmanationRequestDialog extends Application {
 				fromUuid<EffectPF2e>(this.#request.effectUuid),
 				fromUuid<TokenDocumentPF2e>(this.#request.originToken),
 			])
-			if (!spell) throw new Error("resolving spell UUID failed")
-			if (!effect) throw new Error("resolving effect UUID failed")
-			if (!origin) throw new Error("resolving origin UUID failed")
-			if (!origin.actor) throw new Error("origin token has no actor")
+			if (!spell) throw new Error(translate("emanation.dialog.error-spell-uuid"))
+			if (!effect) throw new Error(translate("emanation.dialog.error-effect-uuid"))
+			if (!origin) throw new Error(translate("emanation.dialog.error-origin-uuid"))
+			if (!origin.actor) throw new Error(translate("emanation.dialog.error-token-no-actor"))
 
```

Comment:
No need to translate these errors. They are just there if something goes catastrophically wrong, and if somebody reports that happening, I need to be able to read them.
```suggestion
			if (!spell) throw new Error("resolving spell UUID failed")
			if (!effect) throw new Error("resolving effect UUID failed")
			if (!origin) throw new Error("resolving origin UUID failed")
			if (!origin.actor) throw new Error("origin token has no actor")
```